lib/network/CachedImage.js

```diff
@@ -7,10 +7,16 @@
  *
  * NOTE: Images can also be of type 'data:svg+xml`. This code also works
  *       for svg, but the mipmapping may not be necessary.
+ *
+ * @class CachedImage
```

Comment:
I did read that.

Many things already had `@class` or `@constructor`, so I didn't feel right removing them.

I agree that if they're redundant and do not serve a purpose for ts generation or jsdoc generation, they should be removed.